themes/buildpacks/layouts/shortcodes/comparison-table.html

```diff
@@ -154,16 +193,26 @@
     <tr>
       <th class="p-3">Governance</th>
       <td class="border-left border-right">CNCF</td>
+      <td>Docker</td>
       <td>Red Hat</td>
       <td>Google</td>
       <td>Google</td>
     </tr>
+    <tr class="border-top bg-light">
+      <th class="p-3">Target image types</th>
+      <td class="border-left border-right">Applications</td>
+      <td class="">Applications, Base images, Operating Systems</td>
+      <td class="">Applications</td>
+      <td class="">Applications</td>
+      <td class="">Applications</td>
+    </tr>
     </tbody>
   </table>
 </div>
 <div class="py-4 small">
   <ul class="list-unstyled">
     <li><sup>&dagger;</sup> This feature does not apply to solution.</li>
+    <li><sup>&diams;</sup> Requires additional configuration.</li>
     <li><sup>&ddagger;</sup> Requires additional configuration which adversely affects other features.</li>
   </ul>
```

Comment:
```suggestion
    <li><sup>&dagger;</sup> This feature does not apply to solution.</li>
    <li><sup>&ddagger;</sup> Requires additional configuration which adversely affects other features.</li>
    <li><sup>&diams;</sup> Requires additional configuration.</li>
  </ul>
```

Stylistically, the daggers should stay together. If you'd like to reorder the sentences then I would suggest that the symbols get changed to match.